From 64c68ad42a9089c92dfa4e74f55b6c1b65b8f11c Mon Sep 17 00:00:00 2001
From: Christophe Fergeau <cfergeau@redhat.com>
Date: Sun, 4 Mar 2012 14:32:49 +0100
Subject: [PATCH] Make sure gcc 4.7 can find usleep declaration

Without this, compilation fails with:

libmusicbrainz-4.0.0/src/Query.cc: In member function 'void MusicBrainz4::CQuery::WaitRequest() const':
libmusicbrainz-4.0.0/src/Query.cc:295:19: error: 'usleep' was not declared in this scope
---
 src/Query.cc |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/src/Query.cc b/src/Query.cc
index 5f5934f..9332346 100644
--- a/src/Query.cc
+++ b/src/Query.cc
@@ -33,6 +33,7 @@
 
 #include <string.h>
 #include <sys/time.h>
+#include <unistd.h>
 
 #include <ne_uri.h>
 
-- 
1.7.7.6

